data/json/items/gun/shot.json

```
@@ -853,6 +891,47 @@
     "//": "no provision for separate triggers",
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "shot": 2 } } ]
   },
+  {
+    "id": "srm_1216",
+    "copy-from": "shotgun_pump",
+    "looks_like": "remington_870",
+    "type": "GUN",
+    "name": { "str": "revolver magazine-fed shotgun" },
+    "description": "Lengthy shotgun, that accept rotating 16-round tube magazine.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "srm_1216",
+        "name": { "str": "SRM Arms Model 1216" },
+        "description": "Delayed blowback semi-automatic shotgun, that accept 16-round detachable revolver magazine."
+      }
+    ],
+    "weight": "3062 g",
+    "volume": "8015 ml",
+    "//": "857 x 195 x 97 mm, trimmed about 30% down for empty spaces, minus 3670 ml of magazine volume.",
+    "longest_side": "857 mm",
+    "barrel_length": "470 mm",
+    "price": "1799 USD",
+    "price_postapoc": "50 USD",
+    "to_hit": { "grip": "solid", "length": "short", "surface": "any", "balance": "uneven" },
+    "material": [ "steel", "plastic" ],
+    "dispersion": 425,
+    "sight_dispersion": 500,
+    "durability": 7,
+    "valid_mod_locations": [
+      [ "barrel", 1 ],
+      [ "brass catcher", 1 ],
+      [ "grip mount", 1 ],
+      [ "mechanism", 2 ],
+      [ "muzzle", 1 ],
+      [ "rail mount", 1 ],
+      [ "sights", 1 ],
+      [ "sling", 1 ],
+      [ "underbarrel", 2 ]
```

Why 2 slots for under barrel attachments?

---

i think i copied it from some another gun, i am not very good with gunmods
```suggestion
      [ "underbarrel", 1 ]
```